filebeat/docs/filebeat-options.asciidoc

```diff
@@ -43,6 +43,7 @@ One of the following input types:
     * redis: Reads slow log entries from redis (experimental).
     * udp: Reads events over UDP. Also see <<max-message-size>>.
     * docker: Reads logs from Docker. Also see <<config-containers>> (experimental).
+    * tcp: Reads events over TCP. Also see <<line-delimiter>>, <<max-message-size>> and <<timeout>> (experimental).
```

Comment:
As we don't have separate pages yet for inputs I wonder if we should mark here that this feature is actually experimental.

@dedemorton Thoughts?